.env.example

```diff
@@ -1,3 +1,4 @@
 AFTERPAY_MERCHANT_ID="#####"
 AFTERPAY_SECRET_KEY="***************"
-# AFTERPAY_REGION="AU" # OPTIONAL: available values are AU, CA, NZ, UK, US. If ommitted, defaults to US
+# AFTERPAY_REGION="en-AU" # OPTIONAL: available values are en-AU, en-CA, en-NZ, en-UK, en-US. If ommitted, defaults to en-US
+# AFTERPAY_API_HOSTNAME="global-api-sandbox.afterpay.com" # OPTIONAL: defaults to global-api-sandbox.afterpay.com
```

Comment:
Just thinking that we need not mark them as optional separately and comment them as well.

```suggestion
## Optional variables

AFTERPAY_REGION="en-AU" # OPTIONAL: available values are en-AU, en-CA, en-NZ, en-UK, en-US. If ommitted, defaults to en-US
AFTERPAY_API_HOSTNAME="global-api-sandbox.afterpay.com" # OPTIONAL: defaults to global-api-sandbox.afterpay.com
```